static/js/services/format-date.js

```diff
@@ -19,6 +19,7 @@ var moment = require('moment');
       var config = {
         date: 'DD-MMM-YYYY',
         datetime: 'DD-MMM-YYYY HH:mm:ss',
+        time: 'h:mm A',
```

Comment:
This isn't ideal because it isn't localised or configurable. The other two above are defaulted to these values and updated to the configured values around line 32.

Either:
1) use the translations provided by moment.js (eg: https://github.com/moment/moment/blob/develop/src/locale/en-gb.js#L14 ), or
2) for consistency update the kanso schema and Configuration > Settings > Basic page so people can configure this value.